tools/tools.go

```diff
@@ -1,11 +1,11 @@
+// +build tools
+
```

Comment:
oh man, did mess this up? Is it required to be at the top?

Reply:
from https://golang.org/pkg/go/build/#hdr-Build_Constraints
"near the top of the file, preceded only by blank lines and other line comments. "

bleh, oops, my bad.... bloock comments are not line comments, I suppose.

What was the effect? Is this breaking a build?

Reply:
without moving this to the top `go vet` complained about bad import.